definitions/media-roku_app/golden_metrics.yml

```diff
@@ -0,0 +1,37 @@
+appLaunches:
```

Comment:
There are 6 metrics here - I was unable to find a precedent for more than 5 but I just performed a cursory look, so I'm hoping this is allowed.

From a UI perspective, I wondered about how 6 metrics may look in the UI. I think it will function well because we aren't expecting more than a handful of these Entity types in a given account (a customer is very unlikely to have more than a few Roku channels vs, say, hosts).

Reply:
We aim at 5 at most but I would 6 is fine too.
We can't give any guarantee that those 6 will be shown in all the UI's since some may cut the number lower.

Reply:
Ok, I'm willing to take the risk. We can always decide to trim one off later (right? 😊)